src/providers/VoiceFocusProvider/index.tsx

```diff
@@ -67,8 +67,11 @@ const VoiceFocusProvider: React.FC<Props> = ({
   });
 
   const addVoiceFocus = async (device: Device): Promise<Device | VoiceFocusTransformDevice> => {
+    console.info('Chose a new audio input device', device);
+
     if (voiceFocusDevice) {
       const vf = await voiceFocusDevice.chooseNewInnerDevice(device);
+      console.info('Re-used the same internal state to create an Amazon Voice Focus transform device.');
```

Comment:
For understanding: Do we re-create the audio transform device again or re-use and replace the inner device?